tests/pipelines/lumina/test_lumina_nextdit.py

```diff
@@ -0,0 +1,173 @@
+import gc
+import unittest
+
+import numpy as np
+import torch
+from transformers import AutoModel, AutoTokenizer
+
+from diffusers import AutoencoderKL, FlowMatchEulerDiscreteScheduler, LuminaNextDiT2DModel, LuminaText2ImgPipeline
+from diffusers.utils.testing_utils import (
+    numpy_cosine_similarity_distance,
+    require_torch_gpu,
+    slow,
+    torch_device,
+)
+
+from ..test_pipelines_common import PipelineTesterMixin
+
+
+class LuminaText2ImgPipelinePipelineFastTests(unittest.TestCase, PipelineTesterMixin):
+    pipeline_class = LuminaText2ImgPipeline
+    params = frozenset(
+        [
+            "prompt",
+            "height",
+            "width",
+            "guidance_scale",
+            "negative_prompt",
+            "prompt_embeds",
+            "negative_prompt_embeds",
+        ]
+    )
+    batch_params = frozenset(["prompt", "negative_prompt"])
+
+    def get_dummy_components(self):
+        torch.manual_seed(0)
+        transformer = LuminaNextDiT2DModel(
+            sample_size=128,
+            patch_size=2,
+            in_channels=4,
+            hidden_size=2304,
+            num_layers=24,
+            num_attention_heads=32,
+            num_kv_heads=8,
+            multiple_of=256,
+            ffn_dim_multiplier=None,
+            norm_eps=1e-5,
+            learn_sigma=True,
+            qk_norm=True,
+            cross_attention_dim=2048,
+            scaling_factor=1.0,
+        )
+        torch.manual_seed(0)
+        vae = AutoencoderKL()
+
+        scheduler = FlowMatchEulerDiscreteScheduler()
+        text_encoder = AutoModel.from_pretrained("google/gemma-2b")
```

Comment:
can we use this instead? https://huggingface.co/hf-internal-testing/dummy-gemma

Reply:
oh sorry didn't realize there is only a tokenizer there, will look into making a dummy-genma